# All settings in this file will be uploaded to the parameter server under the project namespace:
# /namespace/params/setting_1 ... setting_n
#
# Parameters declared as <arg name="arg_param" default="param_value"/>
#                          <node name="node_name" pkg="package" type="node_type" ns="namespace" output="screen">
#                            <param name="param" type="double" value="$(arg arg_param)"/>
#
# inside launch files will be uploaded to the parameter server under the node's namespace:
# /namespace/node_name/param
#
# The guideline would be to put commonly used/shared parameters under the namespace and parameters specific to a node
# under the node's namespace.

management_topic: "/management_topic"

controller_server: "controller_node"
database_server: "/cafer_core/db_manager_node"

rgb_topic: "/camera/rgb/image_rect_color"
depth_topic: "/camera/depth_registered/sw_registered/image_rect"
rgb_info_topic: "/camera/rgb/camera_info"
depth_info_topic: "/camera/depth_registered/sw_registered/camera_info"

robot_controller_feedback_topic: "/object_babbling/controller_node/feedback"

motion_detector_topic: "/object_babbling/motion_detector_node"
motion_detector_service: "/object_babbling/motion_detector_node/is_moving"

get_targets_service: "/object_babbling/image_processing_node/get_targets"
set_target_service: "/object_babbling/image_processing_node/set_target"
track_service: "/object_babbling/image_processing_node/track"
workspace_ptcl: "/object_babbling/image_processing_node/workspace_ptcl"
saliency_ptcl: "/object_babbling/image_processing_node/saliency_ptcl"
objects_ptcl: "/object_babbling/image_processing_node/objects_ptcl"

goal_point_topic: "/object_babbling/babbling_node/goal_point_topic"
tracked_point_topic: "/object_babbling/babbling_node/tracked_point_topic"
tracked_object_topic: "/object_babbling/babbling_node/tracked_object_topic"
is_finish_topic: "/object_babbling/babbling_node/is_finish_topic"
